server/db/helpers.js

```diff
@@ -164,4 +165,20 @@ helpers.getInvitationsByRecipient = function(username) {
   });
 };
 
+
+helpers.updateInvitation = function(inviteObj) {
+  return db.Invitation.findOne({
+    where: {'id': inviteObj.id} 
+  }).then(function(invite) {
+    if(!invite) {
+      throw Error('Invitation not found.');
+    }
+    console.log('SINANANANANNA', invite);
```

Comment:
would be helpful to remove extra logging for the next person pulling down changes